exporter/syslogexporter/README.md

```diff
@@ -0,0 +1,111 @@
+# Syslog Exporter
+
+**Stability level**: Alpha
+
+## About The Exporter
+
+The syslog exporter supports sending messages to a remote syslog server.
+
+- This exporter can forward syslog messages to syslog server using [RFC5424][RFC5424] and [RFC3164][RFC3164].
+- It is recommended that this syslog exporter be used with the [syslog_parser][syslog_parser] configured in the receiver.
+  This ensures that all the syslog message headers are populated with the expected values.
+- Not using the `syslog_parser` will result in the syslog message being populated with default header values.
+
+## Configuration
+
+**The following are a few configuration options available to forward syslog messages**:
```

Comment:
> The following are a few configuration options

Does this mean these are not all of them? :sweat_smile:

```suggestion
**The following configuration options are available**:
```

Reply:
changed